IOS/SwiftSeedProject/SwiftSeedProject/Application/ViewModels/LoginViewModel.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  ViewModel.swift
+//  SwiftSeedProject
+//
+//  Created by Brian Sztamfater on 27/3/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Bond
+
+class LoginViewModel {
+    
+    // TODO: Create a service for authentication and remove references to Api
+    private let api: Api
+    private var user: User?
+    
+    var name = Observable<String?>("")
+    var email = Observable<String?>("")
+    
+    init() {
+        // TODO: We need to implement dependency injection
+        self.api = Api.init()
```

Comment:
Super minor, but you write this 
`self.api = Api.init()`
as:
`self.api = Api()`